Http/HttpResponse.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using UnityEngine;
+using UnityEngine.Networking;
+
+namespace DUCK.Http
+{
+	public class HttpResponse
+	{
+		public string Url { get; private set; }
+		public bool Ok { get; private set; }
```

Comment:
`Success`?

Reply:
I used Ok as its common for servers to responde with Ok with status code 200

Reply:
Perhaps use isOk as it's a bool? It's still close to what a server would respond...